argus/exceptions.py

```diff
@@ -38,3 +38,8 @@ class ArgusHeatTeardown(ArgusError):
 class ArgusEnvironmentError(ArgusError):
     """Base class for errors related to the argus environment."""
     pass
+
+
+class ErrorArgusInvalidDecorator(ArgusError):
```

Comment:
shouldn't we keep the same naming convention for argus errors? something like `ArgusInvalidDecoratorError` seems more fit.